queries/markdown/highlights.scm

```diff
@@ -1,9 +1,18 @@
 ;; From MDeiml/tree-sitter-markdown
 [
-  (atx_heading)
+  (atx_heading (heading_content))
```

Comment:
This query needs to be outside the `[..]` to make it work, otherwise it's capturing the whole atx_heading instead of just the heading_content.